kedro/framework/hooks/manager.py

```diff
@@ -83,3 +83,14 @@ def _register_hooks_setuptools(
             len(plugin_names),
             ", ".join(sorted(plugin_names)),
         )
+
+
+class _NullPluginManager:
```

Comment:
I'd suggest adding some doc string explaining what the purpose of the `_NullPluginManager` is for future reference.